features/support/object_model_pages/shop/employee/registration/ee_family_information.rb

```diff
@@ -24,11 +24,11 @@ def self.suffix_dropdown
   end
 
   def self.male_radiobtn
-    'label[for="person_gender_male"] span'
+    'label[for="radio_male"] span'
```

Comment:
let's use a `data-cuke="something"` selector for all new page object model selectors